src/Sri.php

```diff
@@ -87,4 +95,9 @@ private function jsonFilePath(): string
     {
         return config('subresource-integrity.mix_sri_path');
     }
+
+    private function isInEnabled(): bool
+    {
+        return config('subresource-integrity.enabled');
+    }
```

Comment:
Not sure how useful this function is 🤔 
IMHO the config function is clear enough that it won't add any misunderstanding when reading the code.

Reply:
I just copied whats been done here:
https://github.com/Elhebert/laravel-sri/blob/f3f0279fa59b45a0f39e33f65ad762d97a2ccc40/src/Sri.php#L86-L89

Reply:
This was done to avoid long statements in some parts of the code.

You can also find in some parts of the code we used the config helper directly: https://github.com/Elhebert/laravel-sri/blob/master/src/Sri.php#L56.

I also have an issue with how the function is named tbh: `isInEnabled`.

Reply:
Got this